clients/ts/FunctionalTests/FunctionalTests.csproj

```diff
@@ -5,6 +5,18 @@
     <TypeScriptCompileBlocked>True</TypeScriptCompileBlocked>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="selenium\run-ci-tests.ts" />
+    <None Remove="selenium\run-tests.ts" />
+    <None Remove="ts\Common.ts" />
+    <None Remove="ts\ConnectionTests.ts" />
+    <None Remove="ts\HubConnectionTests.ts" />
+    <None Remove="ts\index.ts" />
+    <None Remove="ts\Utils.ts" />
+    <None Remove="ts\WebDriverReporter.ts" />
+    <None Remove="ts\WebSocketTests.ts" />
+  </ItemGroup>
```

Comment:
hmm?

Reply:
It keeps coming back! I don't know what's causing it 😭